### PR TITLE
Deploy from `opentermsarchive.deployment` Ansible collection

### LINKED ISSUE #15: Install Open Terms Archive engine as a dependency
Install the Open Terms Archive engine as a dependency through NPM. This will make declarations runnable without needing an external global executable, and will add versioning support for the engine, enabling deploying breaking changes without breaking unmaintained instances.

Depends on https://github.com/ambanum/OpenTermsArchive/pull/957.

### PR DESCRIPTION
**Pull request by martinratinaud**
Pr reviewed on [contrib instance](https://github.com/OpenTermsArchive/contrib-declarations/pull/748) is applied here

The init script is also chanegd in order to ask for the complementary informations needed.

Fixes #15

### CODE REVIEW

**Comment by martinratinaud** [comment]
@martinratinaud I'd like to generate the fingerprint automatically if I get the server IP but can't remember how to.

Can you please let me know?

File: templates/config/production.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        }
+      }
+    }
+  },
+  "fetcher": {
+    "language": "en"
+  },
+  "notifier": {
+    "sendInBlue": {
+      "updatesListId": ${sendinblueUpdateListId},
```
Comment (Ndpnt) [inline templates/config/production.json:31]: Quotes missing

File: templates/config/production.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      "updateTemplateId": ${sendinblueUpdateTemplateId}
```
Comment (Ndpnt) [inline templates/config/production.json:32]: Quotes missing

**Comment by Ndpnt** [comment]
When I execute `↳ ./init.sh test-instance test-maintainer`, I get:
```
Replacing variables in template files...
$\{instanceName\}: test-instance
$\{instanceMaintainer\}: test-maintainer
sed: RE error: illegal byte sequence
```

**Review by Ndpnt — CHANGES_REQUESTED** [review]

File: templates/config/production.json (hunk 1 above)
Comment (martinratinaud) [reply templates/config/production.json:31]: updateListsId is supposed to be a number, as expected by [the SIB API](https://developers.sendinblue.com/reference/createcontact)

File: templates/config/production.json (hunk 2 above)
Comment (martinratinaud) [reply templates/config/production.json:32]: updateListsId is supposed to be a number, as expected by [the SIB API](https://developers.sendinblue.com/reference/createcontact)

**Comment by martinratinaud** [comment]
> When I execute `↳ ./init.sh test-instance test-maintainer`, I get:
> 
> ```
> Replacing variables in template files...
> $\{instanceName\}: test-instance
> $\{instanceMaintainer\}: test-maintainer
> sed: RE error: illegal byte sequence
> ```

I updated the doc as parameters to this script are not needed anymore.

Anyway, the first line in this bash script is a `read` action, so I'm not sure if you were using the latest version.

Can you please check and let me know?

**Comment by martinratinaud** [comment]
> When I execute `↳ ./init.sh test-instance test-maintainer`, I get:
> 
> ```
> Replacing variables in template files...
> $\{instanceName\}: test-instance
> $\{instanceMaintainer\}: test-maintainer
> sed: RE error: illegal byte sequence
> ```

I updated the doc as parameters to this script are not needed anymore.

Anyway, the first line in this bash script is a `read` action, so I'm not sure if you were using the latest version.

Can you please check and let me know?

**Comment by Ndpnt** [comment]
> > When I execute `↳ ./init.sh test-instance test-maintainer`, I get:
> > ```
> > Replacing variables in template files...
> > $\{instanceName\}: test-instance
> > $\{instanceMaintainer\}: test-maintainer
> > sed: RE error: illegal byte sequence
> > ```
> 
> I updated the doc as parameters to this script are not needed anymore.
> 
> Anyway, the first line in this bash script is a `read` action, so I'm not sure if you were using the latest version.
> 
> Can you please check and let me know?

Ok, it works now.

**Review by Ndpnt — APPROVED** [review]

**Pull request merged**